CodeSniffer/Standards/PSR1/Sniffs/Methods/CamelCapsMethodNameSniff.php

```diff
@@ -34,16 +34,6 @@ class PSR1_Sniffs_Methods_CamelCapsMethodNameSniff extends Generic_Sniffs_Naming
 
 
     /**
-     * Constructs a PSR1_Sniffs_Methods_CamelCapsMethodNameSniff.
-     */
-    public function __construct()
```

Comment:
And this is not change related to PR topic. Please move it to separate PR so that best solution can be discussed separately.

I think, that better approach would be changing parent class constructor to accept 3 given arguments and defaults to currently used values (in parent class).